src/main/java/io/javalin/plugin/openapi/JavalinOpenApi.kt

```diff
@@ -52,6 +55,18 @@ object JavalinOpenApi {
                 }
             }
         }
+
+        val parsedSchema = OpenAPIV3Parser().readContents(Yaml.mapper().writeValueAsString(schema))
+
+        if(parsedSchema.messages.isNotEmpty()){
+            LoggerFactory.getLogger(JavalinOpenApi::javaClass.get()).warn("The generated OpenApi specification is not valid")
```

Comment:
You should store the logger (static, on the class) so you don't have to get a new one each time.

Reply:
Fixed